scripts/leapfrog/README.md

```diff
@@ -0,0 +1,155 @@
+# Leapfrog Documentation
+
+#### Overview
+A Leapfrog upgrade is a major upgrade that skips at least one release. Currently
+RPCO supports leapfrog upgrades from kilo to newton-14.1.
+
+#### Terms
+
+* [RPCO](https://github.com/rcbops/rpc-openstack): Rackspace Private Cloud powered by OpenStack
+* [OSA](https://github.com/openstack/openstack-ansible):  OpenStack Ansible
+* [OSA-OPS](https://github.com/openstack/openstack-ansible-ops):  OpenStack Operations
+* [Kilo](https://github.com/rcbops/rpc-openstack/tree/kilo): The RPCO release of OpenStack Kilo
+* [Newton-14.1](https://github.com/rcbops/rpc-openstack/tree/newton-14.1) The second RPCO release of OpenStack Newton. This version includes Leapfrog upgrade tools.
+
+## Preparing for a lepfrog upgrade
+The start point is a kilo deployment. It is essential to check the kilo
+deployment is healthy before starting the leapfrog process. Many tools are
+available for this, for example tempest.
```

Comment:
The sentencing here is a little clumsy and doesn't read well. I'd suggest keeping it very simple and stating:

* The state of the source environment must be healthy.
* Database  housekeeping should be done to prevent unnecessary database migrations.